### PR TITLE
Added felt-pen & More accurate benchmarking

### PR DESCRIPTION
**Pull request by nin-jin**
```
node .\complex-benchmark.js
chalk          1,201,276 ops/sec
cli-color        135,233 ops/sec
ansi-colors      719,065 ops/sec
kleur          1,404,775 ops/sec
kleur/colors   1,471,413 ops/sec
colorette      1,603,018 ops/sec
felt-pen       1,489,272 ops/sec
nanocolors     1,674,054 ops/sec
```

### CODE REVIEW

**Comment by ai** [comment]
Cool lib!

1. Can you also add your lib to `./test/loading.cjs` and `./test/size.js` benchmarks?
2. Any reasons why `++index` is better? Just to avoid regressions in the future.

**Comment by StoneCypher** [comment]
[This is a stolen library](https://github.com/jorgebucaran/colorette). Ai removed the git history and author, asked people to switch, and when caught, asked the real author to credit him. 

Do not support this person. 

[Please donate to the real one instead](https://github.com/jorgebucaran/colorette).

**Comment by ai** [comment]
@StoneCypher I mentioned `colorette` and add `colorette`’s author to `LICENSE` before any Twitter thread just after `colorette` author asked me.

I am asking `colorette` author for credit because of 2x performance improvements, which he copied. It is OK to copy ideas, but you are right that origin author should be mentioned.

File: test/complex-benchmark.js [hunk 1]
```diff
@@ -17,36 +18,42 @@ function formatNumber(number) {
 }
 
 let suite = new benchmark.Suite()
+let out // eslint-disable-line no-unused-vars
```
Comment (ai) [inline test/complex-benchmark.js:21]: Can we remove it if it is not used?

Comment (ai) [reply test/complex-benchmark.js:21]: Do we need to add `out =` to benchmarks?

**Comment by nin-jin** [comment]
> Any reasons why ++index is better?

It's prevent JIT from inline const string.

**Comment by ai** [comment]
> It's prevent JIT from inline const string.

Wow! Cool idea!

File: test/complex-benchmark.js (hunk 1 above)
Comment (nin-jin) [reply test/complex-benchmark.js:21]: JIT can throw out content of pure functions if result isn't used. It's relevant for simple benchmark but not complex. I will remove it.

**Comment by StoneCypher** [comment]
> @StoneCypher I mentioned `colorette` and add `colorette`’s author to `LICENSE` before any Twitter thread just after `colorette` author asked me.
> 
> I am asking `colorette` author for credit because of 2x performance improvements, which he copied. It is OK to copy ideas, but you are right that origin author should be mentioned.

It's extremely gross that after you were caught ripping off every line, without ever saying that you did anything wrong, you're pointing the finger at him over minor tuning of his own code.

You should have submitted a PR to his library.

**Pull request merged**